mapclientplugins/argonsceneexporterstep/configuredialog.py

```diff
@@ -29,10 +30,15 @@ def __init__(self, parent=None):
         self._workflow_location = None
         self._previousLocation = ''
 
-        self.setWhatsThis("Please read documentation: \nhttps://abi-mapping-tools.readthedocs.io/en/latest/mapclientplugins.argonsceneexporterstep/docs/index.html")
+        self.setWhatsThis('<html>Please read documentation: \n<a href="https://abi-mapping-tools.readthedocs.io/en/latest/mapclientplugins.argonsceneexporterstep/docs/index.html">online help</a></html>')
```

Comment:
I think this text should be `Please read the documentation available <a>here</a> for further details.`